spec.bs

```diff
@@ -3902,6 +3952,10 @@ The <dfn for=Navigator method>updateAdInterestGroups()</dfn> method steps are:
            [=interest group/trusted bidding signals slot size mode=] to |value|.
         1. Otherwise, set [=interest group/trusted bidding signals slot size mode=] to "none".
 
+        <dt>"`maxTrustedBiddingSignalsURLLength`"
+        1. If |value| is a {{long}} and |value| &ge; 0, set |ig|'s [=interest group/max trusted bidding signals url length=] to |value|.

```

Comment:
Wrap the body in a `<dd>`. Same goes for `trustedBiddingSignalsSlotSizeMode` above which is missing this. You can fix that here or in a follow-up. But follow https://github.com/WICG/turtledove/pull/1065/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR3944-R3948 as an example

Reply:
Will fix trustedBiddingSignalsSlotSizeMode in other PRs, since this PR is not mine.